src/cluster/backend/CMakeLists.txt

```diff
@@ -1,14 +1,18 @@
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/zeromq/cmake")
-
-find_package(ZeroMQ)
-
 # Default to building ZeroMQ only if ZeroMQ was found.
 #
 # If a user enabled the cluster backend explicitly (-D ENABLE_CLUSTER_BACKEND_ZEROMQ:bool=ON),
```

Comment:
Nit: that comment seems a bit out of date since the ZeroMQ backend no longer needs explicit enabling but is enabled by default.